### ISSUE TITLE
"Folder icon" can not be found.

### ISSUE THREAD

**Issue by ghuls** [agersant/polaris-web#29]
"Folder" character does not render on Linux (square box)
https://unicode-table.com/en/1F5C0/

Using "File Folder Emoji" character fixes it:
https://unicode-table.com/en/1F4C1/

I tried to find which font on google fonts would contain the U+1F5C0 character, but I was unable to find it.

```diff
diff --git a/tags/browser.tag b/tags/browser.tag
index 07f6cc2..b676389 100644
--- a/tags/browser.tag
+++ b/tags/browser.tag
@@ -399,7 +399,7 @@
                }
 
                .explorerView .directory:before {
-                       content: "🗀";
+                       content: "📁";
                        margin-right: 5px;
                }
 ```

**Comment by agersant** [comment]
Oh, nice catch - I never realized on Windows this was falling back to the system font to work.

However, I think a better solution would be to use the `Folder` icon from the material icon font which is used in the rest of the UI. This would give us a consistent look across OS and I think it looks better than the open file folder.

**Comment by lnicola** [comment]
It works for me, picking up the Symbola (`ttf-symbola`) font.

**Comment by ghuls** [comment]
@lnicola polaris loads the Hind font from google fonts in index.html so it is reasonable to expect that all needed characters can be rendered by that font for the user interface.
```html
<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Hind:300,400,600|Montserrat">
```

**Comment by agersant** [comment]
Fixed in 591445606943069ef31bb1e2f653abe24036debf
